gamecenter/api/helpers/meta.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""API helpers for META info."""
+from flask import current_app
+
+from gamecenter.api.models import Score
+
+
+def get_meta_from_args(args):
+    """Construct the meta dictionary from the args"""
+    meta = {}
+    uri = current_app.config['HOST_URI']
+    meta['total'] = Score.query.count()
```

Comment:
This is a placeholder for now? This depends on what filters the use included in their query request right?